docs/tutorials/functions.ipynb

```diff
@@ -377,8 +377,8 @@
     }
    ],
    "source": [
-    "user_functions = \"gettsim.taxes.abgelt_st\"\n",
-    "user_functions"
+    "functions = \"gettsim.taxes.abgelt_st\"\n",
```

Comment:
Can this become really fool-proof? What are the benefits vs. assuming that string implies a path? Or one could just require explicit import statements? I have not looked at the disambiguation code, but it smells like adding complexity for a little syntactic sugar over

```py
functions = ROOT_DIR / "taxes" / "abgelt_st.py"
```

Reply:
I think it is because you can check whether the path implied by string leads to an existing file. But, when something cannot be parsed, it is not possible to tell whether a path was not correctly parsed or whether an import failed. It will always be the latter error.

I am fine with requiring paths to be `pathlib.Path`s.

Reply:
Yeah, I think I am just a fan of there being exactly one way to do things as opposed to a number of ways. Just makes it harder to explain and maintain.